aspnetcore/mvc/views/tag-helpers/built-in/index.md

```diff
@@ -13,48 +13,48 @@ ms.prod: aspnet-core
 uid: mvc/views/tag-helpers/builtin-th/Index
 ---
 
-# ASP.NET Core Built in Tag Helpers
+# ASP.NET Core Built-in Tag Helpers
 
 By [Peter Kellner](http://peterkellner.net) 
 
-Microsoft has included multiple Tag Helpers that can be immediately used in .net Core web projects.  These Tag Helpers are complete and production quality and are recommended to be used in projects where appropriate.  In this section we will outline all the built in Tag Helpers along with a sample of each tag helper in use.
+Microsoft has included multiple Tag Helpers that can be immediately used in ASP.NET Core projects. These Tag Helpers are feature-complete and of production quality. They are recommended for use in projects wherever appropriate. In this section, we will outline all the built-in Tag Helpers and provide a sample of each.
```

Comment:
The ASP.NET Core framework includes many tag helpers that can help you be more productive writing robust code. Each of the Tag Helpers is documented, complete with sample code.